### ISSUE TITLE
[COGNITION]: Pagination - HTML SHOULD be updated to improve usability for assistive technology

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#9183]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

- **❗️ Must** for if the feedback must be applied 
- **⚠️  Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

Our pagination makes use of a lot of ARIA and tab indexes to mimic regular pagination links. This is less than optimal, and should be updated to use regular `<a>` tags with HREF attributes and client-side route changes where appropriate.

## Related Issues
* https://github.com/department-of-veterans-affairs/va.gov-team/issues/8254
* https://github.com/department-of-veterans-affairs/va.gov-team/issues/8255

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

- [ ] Unit tests are updated and passing
- [ ] No axe errors after HTML is updated
- [ ] No existing functionality is lost
- [ ] Screen readers announce the previous, current, and next page links properly

## Environment

* https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/visual-design/components/pagination/

## Possible Fixes (optional)

```diff
! Pagination.jsx#L53

<a
- aria-label={`Load next page ${this.props.ariaLabelSuffix}`}
+ aria-label={`Next page ${this.props.ariaLabelSuffix}`}
  onClick={() => {this.onPageSelect(this.props.page + 1, 'nav-paginate-next');}}
  onKeyDown={e => this.handleKeyDown(e, this.props.page + 1)}
- tabIndex="0">
  Next
</a>
```

```diff
! Pagination.jsx#L68

<a
- aria-label={`Load previous page ${this.props.ariaLabelSuffix}`}
+ aria-label={`Previous page ${this.props.ariaLabelSuffix}`}
  onClick={() => {this.onPageSelect(this.props.page - 1, 'nav-paginate-previous');}}
  onKeyDown={e => this.handleKeyDown(e, this.props.page - 1)}
- tabIndex="0">
- <abbr title="Previous">Prev</abbr>
+ Prev
</a>
```

```diff
! Pagination.jsx#L165
! https://tink.uk/using-the-aria-current-attribute/

<a
+ aria-current={ currentPage ? 'true' : null }
- aria-label={`Load page ${pageNumber} ${ariaLabelSuffix}`}
+ aria-label={`Page ${pageNumber} ${ariaLabelSuffix}`}
  className={pageClass}
  key={pageNumber}
  onClick={() => this.onPageSelect(pageNumber, 'nav-paginate-number')}
  onKeyDown={e => this.handleKeyDown(e, pageNumber)}
- tabIndex="0">
  {pageNumber}
</a>
```

## WCAG or Vendor Guidance (optional)
* [Link Purpose (In Context): Understanding SC 2.4.4](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-refs.html)
* [Tink: Using the aria-current attribute](https://tink.uk/using-the-aria-current-attribute/)

## Screenshots or Trace Logs
![Screen Shot 2020-05-15 at 4 52 12 PM](https://user-images.githubusercontent.com/934879/82100323-ff5dfb00-96ce-11ea-9998-81503535e39b.png)

**Comment by 1Copenut** [comment]
Issue moved to [department-of-veterans-affairs/vets-design-system-documentation #232](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/232) via [**ZenHub**](https://www.zenhub.com/)
